packages/amplication-server/src/core/moduleDto/moduleDto.service.ts

```diff
@@ -137,8 +157,34 @@ export class ModuleDtoService extends BlockTypeService<
         "AND"
       );
     } else {
-      return [];
+      allDTOs = [];
     }
+
+    const resourceId = args.where.resource?.id;
+    return this.filterDtosByGeneratorName(resourceId, allDTOs);
```

Comment:
@mulygottlieb  await?

Reply:
no need since this is an async function that calls another async function so it returns the promise